src/Resolver/Value/ListValue.php

```diff
@@ -34,17 +34,17 @@ public function getRawValue() : array
         return $return;
     }
 
-    public function printValue() : string
+    public function printValue(bool $prettyPrint) : string
     {
         $component = [];
 
         foreach ($this->value as $value) {
             \assert($value instanceof ValidatedValue);
 
-            $component[] = $value->printValue();
+            $component[] = $value->printValue(true);
```

Comment:
You must pass given parameter, not true... How did it go through tests? You must add test for printing List of Objects.